src/main/java/lotto/domain/PrizeType.java

```diff
@@ -0,0 +1,38 @@
+package lotto.domain;
+
+import lombok.Getter;
+
+@Getter
+public enum PrizeType {
+    FIRST_PRIZE(6, 2000000000),
```

Comment:
숫자를 표현할때 언더바를 이용하면 조금더 가독성 좋게 표현 할 수 있어요! 
```suggestion
    FIRST_PRIZE(6, 2_000_000_000),
```